yaml/morphology/register.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: register
+# TODO Is this an accurate brief for Register?
+brief: relation of utterance to context
```

Comment:
It's OK but I kind of dislike using the names of categories in `brief` or `full` unless the names _actually_ refer to those categories. (Like the way EVERY Function in Ithkuil _except_ Manifestive includes the word, "manifest"!) I can't think of anything better at the moment though.

Reply:
This is why I think we should make a point of capitalizing category and value names. I'll make an issue for that now.